pom.xml

```diff
@@ -164,7 +164,7 @@
         <phoenix.version>4.7.0.2.5.0.0-1245</phoenix.version>
         <redis.lettuce.version>3.4.2.Final</redis.lettuce.version>
         <scala.version>2.10.5</scala.version>
-        <schema-registry.version>0.3.0-SNAPSHOT</schema-registry.version>
+        <schema-registry.version>0.4.0-SNAPSHOT</schema-registry.version>
```

Comment:
Is this needed in this patch? I'd like to make this separate if possible cause upgrading dependency should be easily tracked. If we can't separate this change in the PR, let's merge #1022 in first.

Reply:
I need 0.4.0 Snapshot for this, shell script migration required some code changes in storage tool and that was pushed in 0.4.0 in Schema Registry.